app/controllers/application_controller/ci_processing.rb

```diff
@@ -548,7 +548,7 @@ def evacuatevms
     recs = find_checked_items
     recs = [params[:id].to_i] if recs.blank?
     @record = find_by_id_filtered(VmOrTemplate, recs.first)
-    if @record.is_available?(:evacuate) && !@record.ext_management_system.nil?
+    if @record.supports_evacuate? && !@record.ext_management_system.nil?
```

Comment:
this can be simplified to `if @record.supports_evacuate?` because this already checks for `@record.ext_management_system.nil?`